py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -204,6 +204,8 @@ def check_response(self, response: Dict[str, Any]) -> None:
         if not value:
             value = response['value']
         if isinstance(value, str):
+            if len(value) == 0:
```

Comment:
could we use:

```python
if not value:
    ...
```

here? can't recall if it could be `None` or `""`, also could we add a test for this please?